src/librustdoc/clean/blanket_impl.rs

```diff
@@ -104,10 +104,10 @@ impl<'a, 'tcx> BlanketImplFinder<'a, 'tcx> {
                         // the post-inference `trait_ref`, as it's more accurate.
                         trait_: Some(clean_trait_ref_with_bindings(
                             cx,
-                            ty::Binder::dummy(trait_ref.0),
+                            ty::Binder::dummy(trait_ref.skip_binder()),
```

Comment:
I wonder if this should've been `subst_identity`.

Reply:
Ah thanks! Is it okay to open a tiny PR to just fix both of these? Otherwise I can try to tack it onto something else EarlyBinder-related

Reply:
opening a PR to fix single usesites is fine or even good. We can use a rollup so that they don't block the queue as they aren't perf relevant